data/artists/aparapira-parape.md

```diff
@@ -5,6 +5,7 @@ genres:
   - house
   - disco
   - techno
+image: ./assets/images/APARAPIRA.png
```

Comment:
@marthabanana tze touto to property na valleis stous artists, gia na fainetai i ikona tous sto most recently updated.